activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -791,6 +793,17 @@ def add_foreign_key(from_table, to_table, options = {})
       #
       #   remove_foreign_key :accounts, name: :special_fk_name
       #
+      # The +options+ hash can include the following keys:
```

Comment:
Let's not duplicate all the options but redirect the user to `add_foreign_key` for a list of possible options.